transport/shadowtls/shadowtls.go

```diff
@@ -0,0 +1,138 @@
+package shadowtls
+
+import (
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/binary"
+	"fmt"
+	"hash"
+	"io"
+	"net"
+
+	"github.com/Dreamacro/clash/common/pool"
+	"github.com/Dreamacro/clash/transport/trojan"
+)
+
+const (
+	chunkSize             = 1 << 13
+	MODE           string = "shadow-tls"
+	HASH_LEN       int    = 8
```

Comment:
Camel case is also suggested for consts in Go.  
https://google.github.io/styleguide/go/guide#mixedcaps

Reply:
Will fix

Reply:
Fixed